CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# All changes should be reviewed by a member of the @react-reviewers team
+* @primer/brand-reviewers
```

Comment:
👋 @jonrohan if we add a codeowners file, it should be for a subset of this team. Specifically [this one](https://github.com/orgs/github/teams/primer-brand). I don't think we have this team enabled in the primer org though 🤔. I can create the entitlements for it tomorrow, if you can wait for that.